pkg/iac/deserializers.go

```diff
@@ -53,5 +54,7 @@ func Deserializers() []deserializer.CTYDeserializer {
 		awsdeserializer.NewSNSTopicPolicyDeserializer(),
 		awsdeserializer.NewSNSTopicSubscriptionDeserializer(),
 		awsdeserializer.NewDynamoDBTableDeserializer(),
+
+		ghdeserializer.NewGithubRepositoryDeserializer(),
```

Comment:
Maybe for another PR, we could create 2 different functions to list for the first AWS deserializers and for the second Github ones. And call them here.

I fear that it's gonna be messy in the near future in this function.

Reply:
Maybe, more generally we could think about how we want to handle tf states.

Do we really want to try to decode aws related resources when scanning github ?
But we may want to handle states with mixed providers in the future too ... I think it's too early to take a direction.